Program/23BCA281.py

```diff
@@ -0,0 +1,56 @@
+###[ 7 ] Create table contact(fname,laname,contact,email,city). Perform following operation on it
+##1. Insert Contact 8 records
+##2. Delete Contact¶"""
+
+
+import sqlite3 as sq
+
+#Connect to the py.db databse
+conn=sq.connect("D:\\sqlite\\csv\\py.db")
+cur=conn.cursor()
+#Create table contact
+cur.execute('''
+CREATE TABLE IF NOT EXISTS contact (
+    fname TEXT,
+    lname TEXT,
+    contact TEXT,
+    email TEXT,
+    city TEXT
+)
+''')
+#Insert 8 records in the table
+records = [
+    ('Om', 'Patel', '1234567890', 'om@gmail.com', 'Surat'),
+    ('Sai', 'Patel', '9876543210', 'sai@gmail.com', 'Bardoli'),
+    ('Ram', 'Shah', '4561237890', 'ram@gmail.com', 'Madhi'),
+    ('Bob', 'Brown', '7894561230', 'bob@example.com', 'Vyara'),
+    ('Charlie', 'Johnson', '1231231234', 'charlie@example.com', 'Surat'),
+    ('Diana', 'Williams', '3213213210', 'diana@example.com', 'Mahuva'),
+    ('Eve', 'Davis', '4567891230', 'eve@example.com', 'Vapi'),
+    ('Frank', 'Miller', '6543219870', 'frank@example.com', 'Navsari')
+]
+#Execute insert query
+cur.executemany('''
+INSERT INTO contact (fname, lname, contact, email, city)
+VALUES (?, ?, ?, ?, ?)

```

Comment:
why you write here (?????) question marks ?

Reply:
The question marks "?" in the "executemany" method are placeholders for the actual values which user want to insert into the database and The executemany method is used to insert multiple records into a table at once.

Reply:
please explain in easy way.....

Reply:
> please explain in easy way.....

 Is this use for base on the column here I have five column so I use five time "?".
which means if the user have question marks four column so the user use four times like this(?, ?, ?, ?)